src/lv_objx/lv_kb.h

```diff
@@ -68,6 +68,14 @@ enum {
 };
 typedef uint8_t lv_kb_style_t;
 
+
+enum {
+    LC_KB_SHIFT_LOWER = 0,
```

Comment:
Is there a reason for putting `LC` instead of `LV`?

Reply:
Simple typo.  I've fixed it.